src/python/WMCore/Database/MongoDB.py

```diff
@@ -36,27 +37,48 @@ def __init__(self, database=None, server=None, port=None, replicaset=None,
                       if missing (TODO)
         :logger:      Logger
         """
-        self.server = server # '127.0.0.1'
-        self.port = port # 8230
+        self.server = server
```

Comment:
I can agree about the `server`, but the `port` can be (and even must be) optional, because we may have multiple ports  defined as part of the server strings in the list of replicaset members.  We can indeed address this in the future.

Reply:
Yet I see you made these changes here. Which brings me to the next question now. Is `replicaset` and `port` supposed to be - optionally - part of the keyword arguments now `kwargs`? If so, then can you please update the docstring above saying - no need to be exhaustive, but at least to touch the most common parameters - what could be the extra keyword arguments, their data type and very short description?

Reply:
I can do that yes. I did not do so only because we were not having such practice until now. But, indeed, having them well defined in the MongoClient brings sense to documenting the most common ones here as well. I'll do that with my next commit.